build.gradle

```diff
@@ -10,40 +10,49 @@
 
 buildscript {
     repositories {
-        mavenLocal()
-        //mavenCentral()
-        maven { url "https://repo1.maven.org/maven2" }
-        //maven { url "https://repo.grails.org/grails/core" }
-        maven { url "https://repo.grails.org/grails/plugins/" }
-        maven { url "https://repo.grails.org/grails/plugins-releases/" }
-        maven { url "https://repo.grails.org/grails/core/" }
+        mavenCentral()
     }
     dependencies {
-        classpath "org.grails:grails-gradle-plugin:$grailsVersion"
-        classpath "org.grails.plugins:hibernate5:${gormVersion-".RELEASE"}"
-        classpath "com.bertramlabs.plugins:asset-pipeline-gradle:2.15.1"
-        classpath "gradle.plugin.com.gorylenko.gradle-git-properties:gradle-git-properties:2.2.0"
-        classpath 'org.grails.plugins:database-migration:3.1.0.RC1'
-        classpath "gradle.plugin.com.virgo47.gradle:ClasspathJar:1.0.0"
-
```

Comment:
Oh is this just for the build dependencies?

Reply:
Hi Justin, yeah, it's somewhat confusing when rendered as a diff.

Basically, I used the new [Plugins DSL](https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block) wherever possible, for brevity (it's generally just one line, instead of several scattered about the file in the old `apply plugin:` syntax). I thought this would make merging changes from the v1 branch easier, but I could be wrong.

There are a few cases where I still use the old syntax, especially where the version number is parametrized. You can see this in e.g. the `swaggerCoreVersion` field which applies to both a plugin and a handful of dependencies. Using a variable here should prevent version skew and maybe even the occasional merge bomb.

No dependency was taken away or changed, just refactored.